### ISSUE TITLE
Robo-Nurse Activity Recognition

### ISSUE THREAD

**Issue by semeyerz** [RoboCupAtHome/RuleBook#99]
We have two questions regarding the robo-nurse task:

1. When does the activity of the granny happen?
This means will the robot be able to observe the granny to detect the activity or can it also happen while the robot is fetching the pills which means that the robot would be able to only see the outcome of the activity.

2. Is it a bug that two of the three activities reward only 5 instead of 50 points?

**Comment by LoyVanBeek** [comment]
1. Granny's activity happens after the robot brings the pills and can thus observe Granny.
2. I'm not 100% sure. @kyordhel @balkce ?

**Comment by kyordhel** [comment]
1. The activity may occur at any time after delivering the pills to granny within a reasonable time (aka 30 ~ 120 seconds).

Plus. TC will not wait for the robot to turn arround to see granny (for some robots this is problematic). Think about an automated referee box signaling granny to *die* because robot handled the mints instead of the heart-bitrate-regulator pills and now is having a convulsive behaviour on the floor. The robot should be able to detect the activity also AFTER it occurs.

In other words (common sense) it is fine (at least for me) if granny drops the blanket while the robot is not looking and then the robot realizes the blanket is on the floor. The same happens if granny starts walking (at granny speed i.e. ~3 steps per minute) to the table, robot realizes granny is no longer on chair, locates granny and handles the walking stick.

2. It is.

**Comment by LoyVanBeek** [comment]
Scores are fixed.
For now, I think we should really leave the action to occur when the robot is looking, i.e. after handing the pills. Then you can do Kinect-style action recognition which I at least originally intended. 
Inferring which action took place from the observed changes is very interesting, but really something for next year.
